server/cmd/api/errors.go

```
@@ -1,21 +1,25 @@
 package main
 
 import (
-	"log"
 	"net/http"
 )
 
 func (app *application) internalServerError(w http.ResponseWriter, r *http.Request, err error) {
-	log.Printf("internal server error: %s path: %s error: %s", r.Method, r.URL.Path, err)
+	app.logger.Errorw("internal error", "method", r.Method, "path", r.URL.Path, "error", err.Error())
 	writeJSONError(w, http.StatusInternalServerError, "the server encountered a problem")
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)